lib/src/coap_client.dart

```diff
@@ -129,7 +129,7 @@ class CoapClient {
   /// Sends a GET request.
   Future<CoapResponse> get(
     final String path, {
-    final int accept = CoapMediaType.textPlain,
+    final CoapMediaType accept = CoapMediaType.textPlain,
```

Comment:
One general question I was wondering about: Using the enum value here limits users to the pre-defined values. While CoAP implementations should ignore all unknown Content-Formats, on second thought, maybe this approach is a bit too strict, and we should keep numeric values both as inputs and outputs, using something like this instead:

```suggestion
    final int accept = CoapMediaType.textPlain.numericValue,
```

In CoAP messages, we could add a field for both the numeric value and the parsed `CoapMediaType` (which would return `null` if the value is unknown`).

What do you think, @JosefWN?

Reply:
I think it's fine to constrain the input, it's worth the added type safety, the risk of making mistakes is likely greater than the probability that someone encounters a type that we don't support (and if they do, they could always open a ticket).

Reply:
Sounds reasonable :) Thank you for the feedback!